src/NotificationsApp.vue

```diff
@@ -366,6 +383,10 @@ export default {
 		 * Performs the AJAX request to retrieve the notifications
 		 */
 		async _fetch() {
+			if (this.notifications.length && this.notifications[0].notificationId > this.webNotificationsThresholdId) {
```

Comment:
Will it be the most recent id? Or we should consider iterate and find max value?

Reply:
its sorted `notification_id DESC` by the API